src/coreComponents/physicsSolvers/fluidFlow/CompositionalMultiphaseBase.cpp

```diff
@@ -890,11 +891,19 @@ void CompositionalMultiphaseBase::applySourceFluxBC( real64 const time,
                    "ElementRegions",
                    FieldSpecificationBase::viewKeyStruct::fluxBoundaryConditionString(),
                    [&]( FieldSpecificationBase const & fs,
-                        string const &,
+                        string const & setName,
                         SortedArrayView< localIndex const > const & targetSet,
                         Group & subRegion,
                         string const & )
   {
+    if( fs.getLogLevel() >= 2 && m_nonlinearSolverParameters.m_numNewtonIterations == 0 )
+    {
+      localIndex const numTargetElems = MpiWrapper::sum( targetSet.size() );
+      GEOSX_LOG_RANK_0( GEOSX_FMT(
+                          "CompositionalMultiphaseBase {}: at time {}s, the <{}> boundary condition named \"{}\" is applied to the element set named \"{}\" in subRegion \"{}\". \nThe total number of target elements (including ghost elements) is {}. \nNote that if this number is equal to zero for all subRegions, the boundary condition will not be applied on this element set.",
```

Comment:
One of the advantages of `fmt` is to be able to factorise some error messages.
Please refactor the full call or at least the main error string.

Reply:
Would it be useful to include the actual flux value?

Reply:
I refactored the error messages and factorized them a little more. Now I reuse the error messages in multiple locations, and it seems to work well.

Now, I also output the scale of the boundary condition. It is not exactly the value of the flux if a function is used to get a time-dependent flux, but it is a good start. The actual flux value is hidden in the `applyBoundaryConditionToSystem` and then multiplied by dt, so it is hard to get access to it from the solver. That is another good reason to try to move these log messages inside `FieldSpecificationBase` at some point.